test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -619,6 +619,8 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 
 				for _, cpuCount := range cpuCounts {
 					testpod = startHTtestPod(cpuCount)
+					key := client.ObjectKeyFromObject(testpod)
+					Expect(testclient.Client.Get(context.TODO(), key, testpod)).ToNot(HaveOccurred(), "failed to get pod: %q", key.String())
```

Comment:
This is not harmful but I wonder why is needed because `startHTtestPod` is supposed to return the updated pod, isn't it?

Reply:
Here: https://github.com/openshift/cluster-node-tuning-operator/pull/729#discussion_r1272164393 we can see that although we were suppose to get the updated pod, we didn't (the `nodeName` was empty) The reason for that is still mistiry to me TBH

Reply:
I find counter intuitive and unnecessary to had this code which should be redundant. Can be remove it? If this makes the failure resurface again we have narrowed down the search area